src/ConfigurationClientWrapper.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AppConfigurationClient } from "@azure/app-configuration";
+
+const MaxBackoffDuration = 10 * 60 * 1000; // 10 minutes in milliseconds
+const MinBackoffDuration = 30 * 1000; // 30 seconds in milliseconds
+const MAX_SAFE_EXPONENTIAL = 30; // Used to avoid overflow. bitwise operations in JavaScript are limited to 32 bits. It overflows at 2^31 - 1.
+const JITTER_RATIO = 0.25;
+
+export class ConfigurationClientWrapper {
+    endpoint: string;
```

Comment:
Is this used in any place? Suggest removing it.

Reply:
For future load balance support